experiment/build/gcb_build.py

```diff
@@ -39,84 +40,78 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def _get_buildable_images(fuzzer=None, benchmark=None):
+    return docker_images.get_images_to_build([fuzzer], [benchmark])
+
+
 def build_base_images():
     """Build base images on GCB."""
-    _build(get_build_config_file('base-images.yaml'), 'base-images')
+    image_templates = {'base-image': _get_buildable_images()['base-image']}
+    config = generate_cloudbuild.create_cloud_build_spec(image_templates,
+                                                         build_base_images=True)
+    _build(config, 'base-images')
 
 
 def build_coverage(benchmark):
     """Build coverage image for benchmark on GCB."""
-    coverage_binaries_dir = exp_path.filestore(
-        build_utils.get_coverage_binaries_dir())
-    substitutions = {
-        '_GCS_COVERAGE_BINARIES_DIR': coverage_binaries_dir,
-        '_BENCHMARK': benchmark,
+    buildable_images = _get_buildable_images(benchmark=benchmark)
+    image_templates = {
+        image_name: image_specs
+        for image_name, image_specs in buildable_images.items()
+        if (image_name == (benchmark + '-project-builder') or
```

Comment:
"image_name == (benchmark + '-project-builder')" is added clause, without this coverage build fails.